lib/web/scripts/node-join/install.sh

```diff
@@ -437,6 +438,29 @@ app_service:
     public_addr: ${APP_PUBLIC_ADDR}
 EOF
 }
+# installs the provided teleport config (for database service)
+install_teleport_database_config() {
+    log "Writing Teleport database service config to ${TELEPORT_CONFIG_PATH}"
+    CA_PINS_CONFIG=$(get_yaml_list "ca_pin" "${CA_PIN_HASHES}" "  ")
+    cat << EOF > ${TELEPORT_CONFIG_PATH}
+version: v3
+teleport:
+  nodename: ${NODENAME}
+  auth_token: ${JOIN_TOKEN}
+${CA_PINS_CONFIG}
+  proxy_server: ${TARGET_HOSTNAME}:${TARGET_PORT}
+  log:
+    output: stderr
+    severity: INFO
+auth_service:
+  enabled: no
+ssh_service:
+  enabled: no
+proxy_service:
+  enabled: no
+{{.db_service_section}}
```

Comment:
This might be hard to reason about, as the db service section config is created in a different place than the rest of this config. While this design is sound, it is different from what the rest of the script is doing: i.e. plain string manipulation where variables are simple strings, rather than ready YAML fragments. See my other comment too.